src/lib.rs

```diff
@@ -0,0 +1,198 @@
+//! # Substrate
+//!
+//! Substrate is a Rust framework for building blockchains in a modular and extensible way. While in
+//! itself un-opinionated, it is the main engine behind the Polkadot ecosystem.
+//!
+//! [![github]](https://github.com/paritytech/substrate/) - [![polkadot]](https://polkadot.network)
+//!
+//! This crate in itself does not contain any code and is just meant ot be a documentation hub for
```

Comment:
'just meant to be documentation"

But also, why not re-export the main types and crates used in Substrate through this one? Is that the eventual plan?

Reply:
For FRAME, it is being made here: #14137. I imagine a companion crate called `substrate-client-builder` that is everything else that you might need, see #14268.

Once these two crates are in place, perhaps we can re-export them from this crate. 

As it stands now though, I find it hard to even narrow down what to re-export from this crate.

Reply:
Yes I agree it would be a good idea to re-export `frame` here once it is ready